### PR TITLE
Correct content in https://docs.ipfs.tech/how-to/peering-with-content-providers/

### PR DESCRIPTION
**Pull request by ElPaisano**
# Describe your changes
- [x] Remove NFT.Storage addresses from https://docs.ipfs.tech/how-to/peering-with-content-providers/#protocol-labs table
- [x]  Create table under nft.storage header and remove link to GH repo
- [x]  Remove PLN addresses because they don't work

# Files changed
docs/how-to/peering-with-content-providers.md

# What issue(s) does this address?

Addresses https://github.com/ipfs/ipfs-docs/issues/1394

# Does this update depend on any other PRs?
No

## Checklist before requesting a review
- [x] Passing the beta version of the **Check Markdown links for modified files** check. Action results can be viewed [here](https://github.com/ipfs/ipfs-docs/actions/workflows/action.yml).

## Checklist before merging
- [x] Passing all required checks (The beta **Check Markdown links for modified files** check is not required)

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/how-to/peering-with-content-providers.md [hunk 1]
```diff
@@ -69,7 +69,11 @@ To have your platform added to this list, please [open a PR to edit this page](h
 
 ### NFT.Storage
 
-See [nft.storage/PEERS](https://github.com/nftstorage/nft.storage/blob/main/PEERS).
+| Peer ID | Addresses |
+| ------- | --------- |
+|`12D3KooWCRscMgHgEo3ojm8ovzheydpvTEqsDtq7Vby38cMHrYjt`|`/dns4/nft-storage-am6.nft.dwebops.net/tcp/18402`|
```
Comment (2color) [inline docs/how-to/peering-with-content-providers.md:74]: I tried pinging these unsuccessfully.

**Comment by 2color** [comment]
It seems that all the Protocol Labs ones are not reachable — neither by normal ping or `ipfs ping`. I would remove them all for now.

File: docs/how-to/peering-with-content-providers.md (hunk 1 above)
Comment (ShadowJonathan) [reply docs/how-to/peering-with-content-providers.md:74]: Related: https://github.com/ipfs/ipfs-docs/issues/1429

*all* of the addresses don't seem resolvable

**Comment by ShadowJonathan** [comment]
That means that this PR might "fix" https://github.com/ipfs/ipfs-docs/issues/1429

Though i'd also like to get them replaced with the real addresses

**Comment by ElPaisano** [comment]
@2color @ShadowJonathan removed the PLN addresses https://bafybeiccuxasl32gh52jv7pnnfwipwv4ugzs7q6muuo2kfbl2ig6hhnoti.on.fleek.co/how-to/peering-with-content-providers/#content-provider-list

**Comment by ShadowJonathan** [comment]
i'd recommend to update the PR title as well, then

**Comment by 2color** [comment]
The NFT storage nodes aren't reachable. I'd remove them for now until we have the updated ones.

**Comment by ShadowJonathan** [comment]
A follow-up to this PR would be two issues pinging the respective infrastructure members if they want to add their nodes to this list again.

**Comment by ElPaisano** [comment]
> The NFT storage nodes aren't reachable. I'd remove them for now until we have the updated ones.

Removed

**Review by 2color — APPROVED** [review]

**Pull request merged**